step.yml

```diff
@@ -26,8 +26,13 @@ inputs:
   - plist_path:
     opts:
       title: "Info.plist file path"
+      summary: Path to the given target's Info.plist file. You need to use this step for each archivable target of your project.
       description: |
-        Path to the given project's Info.plist file.
+        **NOTE:**<br/>
+        If your IPA contains multiple build targets ( e.g extensions ), they would need to have the same version number just like your app main target's.<br/>
```

Comment:
can we build the project with different app and extension versions?

Reply:
If your IPA contains multiple build targets ( e.g extensions ), they would need to have the same version number just like your app main target's.<br/>

-> 

`If your IPA contains multiple build targets ( e.g watch app, extensions ), they would need to have the same version number as your app's main target has.<br/>`

Reply:
"can we build the project with a different app and extension versions?"
Yes, you can without any problem.
The problem occurs only during the App Store release

Reply:
Fixed